src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,140 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.25.0.dev3 (01/29/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Allow excludes in BUILD files via new `!` syntax (#9025)
+  `PR #9025 <https://github.com/pantsbuild/pants/pull/9025>`_
```

Comment:
(The prior comment went away due to your change). Bump that this should be a new feature. The API change will be when we deprecate the old behavior.